scripts/manage-repos.sh

```diff
@@ -0,0 +1 @@
+reset-all-repos.sh
```

Comment:
Really nice idea for the interim👌 I guess this can be gradually cleaned up as we do cleanup for having a lot less apps to fetch (like with removing the migrate to uv script)